src/main/java/io/github/thebusybiscuit/slimefun4/api/events/GuideOpenEvent.java

```diff
@@ -0,0 +1,64 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import javax.annotation.Nonnull;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * This {@link Event} is called whenever a {@link Player} tries to open the Slimefun Guide book. 
+ * 
+ * @author Linox
+ *
+ */
+public class GuideOpenEvent extends Event implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private final Player player;
+    private final ItemStack guide;
+    private boolean cancelled;
+
+    public GuideOpenEvent(@Nonnull Player p, @Nonnull ItemStack guide) {

```

Comment:
I think a `SlimefunGuideLayout` with respective getters/setters would be great too